packages/orchestration/src/typeGuards.js

```diff
@@ -126,7 +126,7 @@ export const ICQMsgShape = M.splitRecord(
 export const chainFacadeMethods = harden({
   getChainInfo: M.call().returns(VowShape),
   makeAccount: M.call().returns(VowShape),
-  query: M.call(M.arrayOf(ICQMsgShape)).returns(VowShape),
+  query: M.call(M.arrayOf(M.record())).returns(VowShape),
```

Comment:
it's narrower than that, yes?
```suggestion
  query: M.call(M.arrayOf(TypedJsonShape).returns(VowShape),
```